src/site/content/_redirects.yaml

```diff
@@ -233,3 +233,6 @@ redirects:
 # Redirects for vanity URLs
 - from: /scale-on-web
   to: /tags/scale-on-web
+
+- from: /devices
```

Comment:
hm, I'm a bit concerned with allowing redirects like this because eventually everyone will start doing it. But I understand why it's nice. @kaycebasques do you have an opinion?

Reply:
@kaycebasques (gentle ping)

Reply:
Done.